### PR TITLE
Makes it so wizards can win or halfwin based on objective completion

### PR DESCRIPTION
**Pull request by Amelia0010**
See: https://cdn.discordapp.com/attachments/134720091576205312/754083649380679830/Screenshot_20200911-165843_Firefox_Nightly.jpg

### CODE REVIEW

**Comment by alexkar598** [comment]
I dont think checking for objectives is a good idea, we dont enforce objectives on wizard and wizards dont follow their objectives. I doubt anyone would start following their objectives to see a light on my panel light up

File: code/game/gamemodes/wizard/wizard.dm [hunk 1]
```diff
@@ -64,9 +64,17 @@
 
 /datum/game_mode/wizard/set_round_result()
 	..()
-	if(finished)
+	var/didEscape = TRUE
+	for(var/datum/mind/wizard in wizards)
+		if(!wizard.considered_escaped())
```
Comment (alexkar598) [inline code/game/gamemodes/wizard/wizard.dm]: Proc doesnt exist

**Review by alexkar598 — DISMISSED** [review]

File: code/game/gamemodes/wizard/wizard.dm [hunk 2]
```diff
@@ -64,9 +64,17 @@
 
 /datum/game_mode/wizard/set_round_result()
 	..()
-	if(finished)
+	var/didEscape = TRUE
+	for(var/datum/mind/wizard in wizards)
+		if(considered_escaped(wizard))
+			didEscape = FALSE
```
Comment (jcatjmeow) [inline code/game/gamemodes/wizard/wizard.dm]: You can cast your proc like so
```suggestion
	var/didEscape = TRUE
	var/datum/objective/O
	for(var/datum/mind/wizard in wizards)
		if(O.considered_escaped(wizard))
			didEscape = FALSE
```
This allows you to access /datum/objective/ procs even if you aren't in that file.

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/game_mode.dm [hunk 3]
```diff
@@ -750,4 +750,4 @@
 	round_credits += "<br>"
 	round_credits += "<br>"
 
-	return round_credits
+	return round_credits er
```
Comment (jcatjmeow) [inline code/game/gamemodes/game_mode.dm]:
```suggestion
	return round_credits
```

**Review by jcatjmeow — CHANGES_REQUESTED** [review]

**Comment by SomeguyManperson** [comment]
this only checks for if the wizard escaped, and wizards can get survive until the end instead of escape objectives so they could score a loss even if they completed their objectives

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**